src/mouse.cpp

```diff
@@ -134,8 +134,9 @@ void GraphicsWindow::MouseMoved(double x, double y, bool leftDown,
         double dy = (y - orig.mouse.y) / scale;
 
         if(!(shiftDown || ctrlDown)) {
-            double s = 0.3*(PI/180)*scale; // degrees per pixel
-            if(SS.turntableNav) {          // lock the Z to vertical
+            double sign = SS.cameraNav ? -1.0 : 1.0;
+            double s = 0.3*(PI/180)*scale*sign; // degrees per pixel
+            if(SS.turntableNav) {               // lock the Z to vertical
```

Comment:
And also I think this fixes the `CTRL-middle drag` "hack" of turntable navigation [with this line](src/mouse.cpp) .
https://github.com/solvespace/solvespace/issues/888#issuecomment-759855825
https://github.com/solvespace/solvespace/pull/1307#issuecomment-1304587916